make/common/modules/LauncherCommon.gmk

```diff
@@ -51,7 +51,7 @@ LAUNCHER_CFLAGS += -I$(TOPDIR)/src/java.base/share/native/launcher \
 
 ifeq ($(call isTargetOs, ios), true)
   LAUNCHER_CFLAGS += -I$(TOPDIR)/src/java.base/macosx/native/libjli
-fi
+endif
```

Comment:
Did the original file in mobile have a broken if statement? So this is like a separate bug fix? Otherwise I don't understand what I'm seeing here.

Reply:
That was indeed broken in the mobile build.